operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -367,11 +357,23 @@ Check the expiration of the certificates.
    ```
 
 6. Perform a rolling reboot of master nodes.
-   1. Follow process [Reboot_NCNs](../node_management/Reboot_NCNs.md)
 
-   **IMPORTANT:** Please ensure you are verifying pods are running on the master node that was rebooted before proceeding to the next node.
+   1. Follow the [Reboot_NCNs](../node_management/Reboot_NCNs.md) process.
+
+      **IMPORTANT:** Please ensure you are verifying pods are running on the master node that was rebooted before proceeding to the next node.
 
 7. Perform a rolling reboot of worker nodes.
-   1. Follow process [Reboot_NCNs](../node_management/Reboot_NCNs.md)
 
-   **IMPORTANT:** Please keep track of where the nexus pod is running and ensure it is either scaled down to 0 replicas or has moved and full started via the drain process.
+   1. Before rebooting any worker node, scale nexus replicas to 0.
+        
+   ```bash
+   ncn-m# kubectl scale deployment nexus -n nexus --replicas=0
+   ```
+
+   2. Follow the [Reboot_NCNs](../node_management/Reboot_NCNs.md) process.
+
+   3. After rebooting all the worker nodes, scale nexus replicas back to 1.
+        
+   ```bash
+   ncn-m# kubectl scale deployment nexus -n nexus --replicas=1
+   ```
```

Comment:
Is there any validation that the admin should do after following this process?

Reply:
I am removing the nexus scale from main, 1.2, 1.1 and 1.0 branches for the cert renewal doc and only keeping it in release/0.9 which is what also covers Shasta v1.3  I confirmed with Bob that nexus only needs to be scaled down prior to reboot in Shasta v1.3